src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -187,6 +212,49 @@ await featureFilter.EvaluateAsync(context).ConfigureAwait(false) == targetEvalua
             return enabled;
         }
 
+        private void BindSettings(IFeatureFilterMetadata filter, FeatureFilterEvaluationContext context)
+        {
+            IFilterParametersBinder binder = filter as IFilterParametersBinder;
+
+            if (binder == null)
+            {
+                return;
+            }
+
+            object settings;
+
+            ConfigurationCacheItem cacheItem;
+            
+            //
+            // Check if settings already bound from configuration or the parameters have changed
+            if (!_cache.TryGetValue(context.FeatureName, out cacheItem) ||
+                cacheItem.Parameters != context.Parameters)
+            {
+                settings = binder.BindParameters(context.Parameters);
+
+                if (_options.FilterSettingsCacheTtl > TimeSpan.Zero)
+                {
+                    _cache.Set(
+                        context.FeatureName,
+                        new ConfigurationCacheItem
+                        {
+                            Settings = settings,
+                            Parameters = context.Parameters
+                        },
+                        new MemoryCacheEntryOptions
+                        {
+                            AbsoluteExpirationRelativeToNow = _options.FilterSettingsCacheTtl
```

Comment:
`SlidingExpiration` continues to slide if it's used, right? So, if it was set to 30 seconds and a user kept refreshing / hitting the same endpoint within 30 seconds. It'd be used indefinitely (unless something else expired it). 

If I understand that correctly, then I could see using it here if we pick a very small value. Like 1 second, so it lasts throughout whatever computation they are doing. 

Because we're enabling developers to pick the cache duration, I'd rather stick with `AbsoluteExpirationRelativeToNow`. Because I think its functionality is what they'd expect.

Do you think `SlidingExpiration` might make more sense here?

Reply:
I misunderstood. You're right, this should be absolute expiration!